scripts/build.js

```diff
@@ -30,6 +31,23 @@ const runWebpack = config => {
   });
 };
 
+const compileAll = async () => {
+  webpackConfigs.forEach(async config => {
+    await runWebpack(config);
+  });
+};
+
+const cleanDistFolders = async () => {
+  builds.forEach(async ({ paths }) => {
```

Comment:
For this function to wait until all async work is done, it would need to do something like this:

```js
await Promise.all(
  builds.map(async () => {
    ...
  })
);
```